routes/tests/shopifyAuth.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require('supertest');
+const http = require('http');
+const express = require('express');
+
+const {MemoryStrategy} = require('../../strategies');
+const createShopifyAuthRouter = require('../shopifyAuth');
+
+let server;
+describe('shopifyAuth', async () => {
+  afterEach(() => {
+    server.close();
+  });
+
+  describe('/', () => {
+    it('responds to get requests by returning a redirect page', async () => {
+      const {app, server} = await createServer();
+      const {status, text} = await request(app).get('/?shop=shop1').then((resp) => resp);
+
+      expect(status).toBe(200);
+      expect(text).toMatchSnapshot();
+
+      server.close();
```

Comment:
I think we might want to remove the locally scoped `server` given the `afterEach` block.